bids2openminds/main.py

```diff
@@ -312,11 +312,63 @@ def create_subjects(subject_id, layout_df, layout, collection):
     return subjects_dict, subject_state_dict, subjects_list
 
 
+def create_file_bundle(path, collection, parent_file_bundle=None):
+
+    name = str(path).replace("/", "_").replace("\\", "_")
+    if name[0] == "_":
+        name = name[1:]
```

Comment:
reduce to "sub-01_ses-01_anat" (not whole path)

Reply:
this example would mean that there is also a filebundle for "sub-01" and "sub-01_ses-01"